sw/device/tests/crypto/cryptotest/json/aes_commands.h

```diff
@@ -0,0 +1,60 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_TESTS_CRYPTO_CRYPTOTEST_JSON_AES_COMMANDS_H_
+#define OPENTITAN_SW_DEVICE_TESTS_CRYPTO_CRYPTOTEST_JSON_AES_COMMANDS_H_
+#include "sw/device/lib/ujson/ujson_derive.h"
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define AES_CMD_MAX_MSG_BYTES 64
```

Comment:
Out of curiosity, where does this limit come from?

Reply:
It's arbitrary. uJSON just needs to know at compile-time how big to make the array. If none of our testvectors are that big, we can decrease this and improve the performance a bit.